### PR TITLE
[BULK] DocuTune - Fix build validation issues: docs-link-absolute

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: entity-framework/core/providers/cosmos/planetary-docs-sample.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ Your feedback is valuable! [File an issue](https://github.com/dotnet/EntityFrame
 
 ## Introducing Planetary Docs
 
-You may (or may not) know that Microsoft's [official documentation](https://docs.microsoft.com) runs entirely on open source. It uses markdown with some metadata enhancements to build the interactive documentation that .NET developers use daily. The hypothetical scenario for Planetary Docs is to provide a web-based tool for authoring the docs. It allows setting up the title, description, the alias of the author, assigning tags, editing markdown and previewing the HTML output.
+You may (or may not) know that Microsoft's [official documentation](/) runs entirely on open source. It uses markdown with some metadata enhancements to build the interactive documentation that .NET developers use daily. The hypothetical scenario for Planetary Docs is to provide a web-based tool for authoring the docs. It allows setting up the title, description, the alias of the author, assigning tags, editing markdown and previewing the HTML output.
```
Comment (annmariehitchcock) [inline entity-framework/core/providers/cosmos/planetary-docs-sample.md:102]: Link is missing. I believe it should remain (https://docs.microsoft.com/)

**Comment by annmariehitchcock** [comment]
#docutune-wait

File: samples/end2end/PlanetaryDocs/README.md [hunk 2]
```diff
@@ -35,10 +35,10 @@ git clone https://github.com/dotnet/EntityFramework.Docs
 
 ### Create an Azure Cosmos DB instance
 
-To run this demo, you will need to either run the [Azure Cosmos DB emulator](https://docs.microsoft.com/azure/cosmos-db/local-emulator)
+To run this demo, you will need to either run the [Azure Cosmos DB emulator](/azure/cosmos-db/local-emulator)
```
[the remark this answers is not in the document]
Comment (alexbuckgit) [reply samples/end2end/PlanetaryDocs/README.md]: Fixed. Thanks for the feedback!

**Comment by alexbuckgit** [comment]
#docutune-ready

**Comment by alexbuckgit** [comment]
@ajcvickers @bricelam @roji

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

**Review by ajcvickers — APPROVED** [review]

**Pull request merged**